scripts/yamale.bug

```diff
@@ -0,0 +1,59 @@
+%YAML 1.1
+---
+id: b96bf67
+title: Incorrect orientation (wrong frame) passed to set_attitude_target
+description: >
+     MAVROS operates on a frame of base_link->ENU, whereas MAVLink
+     operates on aircraft->NED. Therefore, location and orientation
+     must be transformed between these two frames when communicating
+     between them.  The buggy function correctly performs the
+     necessary frame conversions, but fails to pass the converted
+     orientation in the "set_attitude_target" function call.
+classification: CWE-137 Representation Errors #PHYSICAL
+keywords:
+  - incorrect argument
+  - function call
+  - frame conversion
+system: mavros
+severity: error
+links: []
+bug:
+  phase: runtime-operation
+  specificity: general issue
+  architectural-location: N/A
+  application: N/A
+  task: N/A
+  subsystem: N/A
+  package: [mavlink/mavros/mavros]
+  languages:
+    - C++
+  detected-by: developer
+  reported-by: contributor
+  issue: N/A
+  time-reported:
+  reproducibility: N/A
+  trace: N/A
+fix:
+  repo: https://github.com/mavlink/mavros
+  hash: b96bf672a718b9f0c9694e4314283e385ba96231
+  pull-request: https://github.com/mavlink/mavros/pull/537
+  license:
+    - BSD
+    - GPLv3
+    - LGPLv3
+  fix-in:
+    - mavros/src/plugins/setpoint_raw.cpp
+  languages:
+    - C++
+  time: 2016-04-29T13:59:23-04:00
+time-machine:
+  ros_distro: indigo
+  ros_pkgs:
+    - mavros
+  datetime: 2016-04-29T13:59:23-04:00
+bugzoo:
+  is-build-failure: false
+  bug-commit: 4f317228c5d5c5bfc96f3f0dfa692bc4b93dcc43
+  fix-commit: b96bf672a718b9f0c9694e4314283e385ba96231
+  fork-urls:
+    - https://github.com/robust-rosin/mavros
```

Comment:
This is a "test bug", correct? Should we rename it to something like `yamale_test.bug`?

Reply:
We could even delete this. I haven't been careful with this one. The kobuki bug in this PR is more complete.